templates/profile-edit.html

```diff
@@ -52,8 +52,7 @@ <h2>Manage email addresses</h2>
                       type="submit"
                       name="email_delete"
                       value="<%= thisEmail.email %>"
-                      class="cd-button cd-button--small cd-button--bold cd-button--uppercase cd-button--danger"
-                      onclick="return confirmDeletion('<%= thisEmail.email %>')"
```

Comment:
The PR is basically replacing this one line containing `onclick` for CSP-friendly code. (second instance)